src/plusplus.coffee

```diff
@@ -33,7 +33,9 @@ module.exports = (robot) ->
     # from beginning of line
     ^
     # the thing being upvoted, which is any number of words and spaces
-    ([\s\w'@.-]*)
+    ([\s\w'@.-:]*)
```

Comment:
Including the : within the matching group also opens up the possibility of giving karma to slack emoji, which seems like a big plus. :metal:++